src/app/features/swap-form/reducer.ts

```diff
@@ -1,10 +1,14 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
-import { AMM, Position } from '@voltz-protocol/v1-sdk';
+import { AMM, InfoPostSwapV1, Position } from '@voltz-protocol/v1-sdk';
+import { Signer } from 'ethers';
+// eslint-disable-next-line no-restricted-imports
+import { WritableDraft } from 'immer/dist/internal';
```

Comment:
we should not be doing this. There is reason why you should never import from node modules directly. Can you check if it is possible to get this import from another place? 

Use: https://github.com/immerjs/immer/issues/726 it seems a type called `Draft` is exported from `@redux/toolkit`

Reply:
[c0724b9](https://github.com/Voltz-Protocol/voltz-ui/pull/572/commits/c0724b9b2c4478849f20f00b94dc25a7f70eb2ae)